people-and-planet-ai/image-classification/requirements.txt

```diff
@@ -1,4 +1,4 @@
 pillow==9.2.0
 apache-beam[gcp]==2.37.0
```

Comment:
@davidcavazos can we upgrade this to `2.43.0` then?

Reply:
Yes, that should be safe to do. It might pull in additional dependency constraints, but hopefully easy ones to solve.

Reply:
Thank you! Going to try this..

Reply:
Seems like 2.42 is not usable yet
```
    The user requested google-cloud-bigquery==3.3.5
    apache-beam[gcp] 2.42.0 depends on google-cloud-bigquery<3 and >=1.6.0; extra == "gcp"
```
there is `2.43.0rc1` or `2.43.0rc2` that could be used, but I'll wait for `2.43.0` full release to be out.

Reply:
FYI, updating requirements.txt and retrying